CoreLayersView/src/au/gov/asd/tac/constellation/views/layers/LayersViewController.java

```diff
@@ -84,15 +75,108 @@ public void submit() {
         if (pane == null) {
             return;
         }
-
         final List<String> layerQueries = new ArrayList<>();
-        for (Node node : pane.getLayers().getChildren()) {
-            if (GridPane.getRowIndex(node) > 0 && GridPane.getColumnIndex(node) == 2) {
-                final TextArea queryTextArea = (TextArea) node;
-                layerQueries.add(queryTextArea.getText().isBlank() ? null : queryTextArea.getText());
+        for (LayerDescription layer : pane.getlayers()) { // TODO: This was the last change of tuesday
```

Comment:
I think only you will understand that TODO @aldebaran30701. Can you either elaborate or we get rid of it?

Reply:
Good catch - Should start searching the project for TODO: before committing